src/models/users_model.js

```diff
@@ -164,6 +164,28 @@ module.exports.getUserInfo = (client, collection, email, callback) => {
   );
 };
 
+module.exports.changeUserInfo = (client, collection, email, password, name, callback) => {
```

Comment:
I think when changing any user info, the user should also be entering in their current password.